bbq/javatests/java/com/google/bbq/QueryResponseSenderTest.java

```diff
@@ -24,56 +26,87 @@
 import android.support.annotation.NonNull;
 import com.google.bbq.Protobufs.BroadcastQuery;
 import com.google.bbq.Protobufs.BroadcastQueryResponse;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
+
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
```

Comment:
There seem to be a few unused import statements with these changes

Reply:
I really need to create a checkstyle config for the tests. The main reason why it doesn't currently look at test code is that I don't want to force some things, like Javadoc on all public methods, on test code where it's not useful.